### ISSUE TITLE
Demo site for InvoiceShelf

### ISSUE THREAD

**Issue by rihards-simanovics** [InvoiceShelf/InvoiceShelf#127]
**Is your feature request related to a problem? Please describe.**
Not a problem just a suggestion, people may want to see what the software looks like and what features it has before invested time in setting one up.

**Describe the solution you'd like**
A simple demo site, some Vue template mod. is required. We also need a timer for reset - best if for each user and limited to something like 5 minutes.

**Describe alternatives you've considered**
Snapshots with reset, but they are not good as they can get easily abused by bad actor.

**Comment by AverageHelper** [comment]
There seems to be an instance at https://demo.invoiceshelf.com, but idk the credentials..

**Comment by rihards-simanovics** [comment]
looks like @gdarko is cooking something.

**Comment by gdarko** [comment]
> looks like @gdarko is cooking something.

Yeah, trying to set up the permissions in a way that will not allow harmful actions :-)

**Comment by OniriCorpe** [comment]
Maybe just disabling email sending would be sufficient against bad usages

**Comment by rihards-simanovics** [comment]
that is certainly an option, but the problem is we need to rework the frontend & backend to do that.

**Comment by l1ghtnings** [comment]
hi guys, do you have login details where I can test the demo?

**Comment by rihards-simanovics** [comment]
Reopening as I'm building a better solution. Keep your eyes pilled on status updates here.

**Comment by gdarko** [comment]
Hi @rihards-simanovics  

The demo is set up but unsure what happened to the credentials. I believe i haven't properly put it into demo mode.

I am checking it now

**Comment by rihards-simanovics** [comment]
> Hi [@rihards-simanovics](https://github.com/rihards-simanovics)
> 
> The demo is set up but unsure what happened to the credentials. I believe i haven't properly put it into demo mode.
> 
> I am checking it now

Yeah, no worries, this one, of course, can stay. I am setting up a new demo that allows users to have a personal instance for approximately 20 minutes. After this time, the demo shuts down and removes the data. The demo will be available here: https://demo.invoiceshelf.os.griffin-web.studio/. I am testing to ensure that SMTP and mail are disabled.

**Comment by rihards-simanovics** [comment]
This is what it might look like (I set it for 1min here, but the reality would be more like 10~15 min). Each instance is dedicated, so a new docker image is spun up so you don't need to worry about people changing admin passwords as such or resets as docker lean it up automagically: 

https://github.com/user-attachments/assets/a46f97b0-9dba-4d0c-8872-321320312505

**Comment by rihards-simanovics** [comment]
The demo site is now live on my new organization demo server. The server is located in the EU, so connection speeds may vary depending on your location.

# When asked what Database to use select SQLite
and keep all of the settings by default.

From the time you start interacting with the demo, a personal instance will be assigned to you. You'll have 10 minutes before the instance expires; after this time, you'll need to restart.

**Comment by rihards-simanovics** [comment]
# ~~Access the new Demo site via this link: https://demo.invoiceshelf.os.griffin-web.studio/~~

~~It goes without saying that you should not enter any real personal details. Treat this as a public space.~~

I am no longer part of the IS team and this demo is no longer hosted.
